components/gateway/Caddyfile

```diff
@@ -101,13 +105,33 @@
 	}
 }
 
+(audit) {
```

Comment:
Should we add it also for /versions and all other api calls returning 502 ?

Reply:
I think we should log every requests.
@flemzord ?

Reply:
All calls must be logged